pkg/cmd/pulumi/pulumi.go

```diff
@@ -286,6 +286,7 @@ func NewPulumiCmd() *cobra.Command {
 		"Run pulumi as if it had been started in another directory")
 	cmd.PersistentFlags().BoolVarP(&cmdutil.Emoji, "emoji", "e", runtime.GOOS == "darwin",
 		"Enable emojis in the output")
+	cmd.PersistentFlags().BoolVarP(&cmdutil.FullyQualifyStackNames, "fully-qualify-stack-names", "Q", false, "Show fully-qualified stack names")
```

Comment:
It would be really nice to be able to set this in some sort of  `.pulumirc`, but we don't have the functionality yet.  Maybe a hackathon project for another time :thinking: